public/app/features/plugins/admin/components/PluginUsage.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from '@emotion/css';
+import React, { useMemo } from 'react';
+import { useAsync } from 'react-use';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { of } from 'rxjs';
+
+import { GrafanaTheme2, PluginMeta } from '@grafana/data';
+import { config } from '@grafana/runtime';
+import { Alert, Spinner, useStyles2 } from '@grafana/ui';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import { SearchResultsTable } from 'app/features/search/page/components/SearchResultsTable';
+import { getGrafanaSearcher, SearchQuery } from 'app/features/search/service';
+
+type Props = {
+  plugin: PluginMeta;
+};
+
+export function PluginUsage({ plugin }: Props) {
+  const styles = useStyles2(getStyles);
+
+  const searchQuery = useMemo(() => {
+    return {
+      query: '*',
+      panel_type: plugin.id,
+      kind: ['panel'],
+    } as SearchQuery;
+  }, [plugin]);
+
+  const results = useAsync(() => {
+    return getGrafanaSearcher().search(searchQuery);
+  }, [searchQuery]);
+
+  const found = results.value;
+  if (found?.totalRows) {
+    return (
+      <div className={styles.wrap}>
+        <div className={styles.info}>
+          {plugin.name} is used <b>{found.totalRows}</b> times.
+        </div>
+        <AutoSizer>
+          {({ width, height }) => {
+            return (
+              <SearchResultsTable
+                response={found}
+                width={width}
+                height={height}
+                clearSelection={() => {}}
+                keyboardEvents={of()}
+                onTagSelected={() => {}}
+              />
+            );
+          }}
+        </AutoSizer>
+      </div>
+    );
+  }
+
+  if (results.loading) {
+    return <Spinner />;
+  }
+
+  if (!config.featureToggles.panelTitleSearch) {
```

Comment:
This is possible if people use the URL directly -- but it will not be an option in the UI!